pyproject.toml

```diff
@@ -63,7 +63,8 @@ docs = [
     "mkdocs-gen-files",             # Plugin for generating additional files in mkdocs
     "mkdocs-literate-nav>=0.6.1",   # Plugin for literate-style navigation in mkdocs
     "mkdocs-gallery",               # Plugin for adding image galleries to mkdocs
-    "mkdocs-material",
+    "mkdocs-material",              # Material theme for mkdocs
+    "plotly",                       # Plotly library for interactive visualizations (used in config)
```

Comment:
why is plotly added? none of the examples look interactive.

Also, unsure why the Github interface is highlighting everything in red